components/layout/segment/segment.js

```diff
@@ -26,7 +32,18 @@ export default function Segment(props) {
                 </div>
             </div>
             <div className={segmentStyle.graph}>
-                {props.graph}
+                {/* if doubleGraph is true (or essentially there is more than one graph, then it will conditionally render with a new div) */}
+                {doubleGraph ? 
+                props.graph.props.children.map((graph,i) => {
+                    return(
+                    <div className={styles.graphContainer} key={i}> 
+                        {graph}
+                    </div> 
+                )})
+                
+                : props.graph
```

Comment:
Can we swap out the ternary operator for an if statement? I think that would be better for readability

Reply:
yeah sure, it would be good for people to know that tho

Reply:
It’s great to know but the syntax gets a bit messy when split across multiple lines